src/lib/dom.js

```diff
@@ -237,11 +237,14 @@ export function extendUrl(url : string, params : { [key : string] : string } = {
 }
 
 export function redirect(win : CrossDomainWindowType = window, url : string) : ZalgoPromise<void> {
-    return new ZalgoPromise(resolve => {
+    return new ZalgoPromise((resolve, reject) => {
 
         info(`redirect`, { url });
 
         setTimeout(() => {
+            if (url.toLowerCase().replace(/[^a-z:]+/g, '').match(/^javascript:/)) {
```

Comment:
Just a nit pick, can we move this logic outside of this setTimeout?

Reply:
I also re-tested this and it worked correctly! https://github.com/paypal/paypal-checkout-components/pull/2371/commits/5f7433272bb44aa9246516d0c11e525e92d4b69b